extra/onnx_ops.py

```diff
@@ -263,3 +266,75 @@ def OneHot(indices, depth, values, axis=-1):
 
 def Floor(x:Tensor): return x.floor()
 def Ceil(x:Tensor): return x.ceil()
+
+def EmbedLayerNormalization(input_ids, segment_ids:Optional[Tensor]=None, word_embedding:Tensor=None, position_embedding:Tensor=None, segment_embedding:Optional[Tensor]=None, gamma=None, beta=None, mask:Optional[Tensor]=None, position_ids:Optional[Tensor]=None, epsilon=None, mask_index_type=None):
+  # https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md#com.microsoft.EmbedLayerNormalization
+  # _type_constraints(input_ids, segment_ids, mask, position_ids, dtypes=[dtypes.int32])  # T1
+  # _type_constraints(word_embedding, position_embedding, segment_embedding, gamma, beta, dtypes=[dtypes.float, dtypes.float16])  # T
+  assert (segment_ids is None and segment_embedding is None) or (segment_ids is not None and segment_embedding is not None)
```

Comment:
`(segment_ids is None) is (segment_embedding is None)`

Reply:
Neat 👍 [2828faf](https://github.com/tinygrad/tinygrad/pull/1537/commits/2828fafe1e5cb3638fc4d1902a30294de94c1323)